src/Components/WebAssembly/WebAssembly/src/Services/HttpClientServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
```

Comment:
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
using System;
```

(Sorry, should have mentioned this in my original suggestion)

Reply:
Don't worry, we all missed that one 😅.